### PR TITLE
Use Makefile instead of setup.py for install

### LINKED ISSUE #13: Switch away from setup.py
Directly running setup.py is deprecated and it produces some DeprecationWarning.  Unfortunately pip-based installation seems not possible because pip cannot create files in /usr/lib/lsb.

Maybe we can use pip-based installation and symlink the needed files to /usr/lib/lsb with explicit book instructions?  Or just use a plain Makefile to install the package?

### PR DESCRIPTION
**Pull request by xry111**
Directly running setup.py is deprecated.  As this package is simple enough, use a plain Makefile as the building system.

Fixes #13.

### CODE REVIEW

**Comment by renodr** [comment]
I just tested this and it looks good to me. I think that can be merged once @pierre-labastie reviews it, and we should probably push out a new release too since we're close in proximity to package freeze.

**Review by pierre-labastie — CHANGES_REQUESTED** [review]
I think, we should address what is in the two comments before merging

File: man/install_initd.8 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
-.TH INSTALL_INITD "8" "July 2022" "install_initd (LSB-Tools-0.10)" "LSB-Tools"
+.TH INSTALL_INITD "8" "February 2024" "install_initd (LSB-Tools-0.11)" "LSB-Tools"
```
[the remark this answers is not in the document]
Comment (xry111) [reply man/install_initd.8:2]: But all other man pages in LFS/BLFS contain versions.  Say, `man ld` and `man passwd`.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,78 @@
+Q = @
```
[the remark this answers is not in the document]
Comment (xry111) [reply Makefile:2]: Done.

**Pull request merged**